lexicons/app/bsky/actor/defs.json

```diff
@@ -134,6 +134,16 @@
           }
         }
       }
+    },
+    "personalDetailsPref": {
+      "type": "object",
+      "properties": {
+        "birthDate": {
+          "type": "string",
+          "format": "datetime",
+          "description": "The birth date of the owner of the account."
+        }
```

Comment:
What will our convention be for using datetimes to store a date?

Reply:
🤷 do we need to worry about it? Time can be anything

Reply:
Yeah, I do think it's worth considering, have totally run into issues around treating dates as datetimes.  Mostly because folks truly mean a date, which specifically throws away time and timezone.

For example, if my birthday is `1980-05-30`, there's no situation where it should be interpreted as `1980-05-29`, even though the moment I was born could have been `1980-05-29` in some areas of the world.  (Maybe underlined even more by the fact that folks wont be inputting their exact birthtime.)

Two options here are a. represent it at midnight GMT, b. always ignore time information, take the `yyyy-mm-dd` part of the datetime verbatim.

Reply:
Oh yeah that's all legit, my point is I'm not sure we need to be super precise with this because we just need to calculate user age